src/components/MeasureRoutes/MeasureRoutes.test.tsx

```diff
@@ -0,0 +1,153 @@
+import "@testing-library/jest-dom";
+// NOTE: jest-dom adds handy assertions to Jest and is recommended, but not required
+
+import * as React from "react";
+import { render } from "@testing-library/react";
+import MeasureBrowserRoutes, { MeasureRoutes } from "./MeasureRoutes";
+import { MemoryRouter } from "react-router";
+import { ApiContextProvider, ServiceConfig } from "../../api/ServiceContext";
+import { describe, expect, test } from "@jest/globals";
```

Comment:
No clue why, but importing `expect` causes type errors:
```TS2339: Property 'toBeInTheDocument' does not exist on type 'Matchers<void, HTMLElement>'.```

Removing the import doesn't seem impact the test execution and resolves the type errors. 🤷